go/common/entity/entity.go

```diff
@@ -77,6 +77,11 @@ func Load(baseDir string, signerFactory signature.SignerFactory) (*Entity, signa
 		return nil, nil, err
 	}
 
+	if ent.ID != signer.Public() {
+		signer.Reset()
+		return nil, nil, fmt.Errorf("public key mismatch\nsigner: %s\nentity: %s", signer.Public(), ent.ID)
```

Comment:
Don't use newlines in error messages, do something like:
```suggestion
		return nil, nil, fmt.Errorf("public key mismatch (signer: %s entity: %s)", signer.Public(), ent.ID)
```

Reply:
Hm, this still uses newlines?

Reply:
Oh. How embarrassing. I thought I committed this suggestion.